lib/libevent/libevent.nit

```diff
@@ -484,6 +535,28 @@ class ConnectionFactory
 		bev.setcb conn
 	end
 
+	# Accept a connection on a Unix socket
+	fun accept_unix_connection(listener: ConnectionListener, fd: Int, addrin: Pointer, socklen: Int)
+	do
+		var base = listener.base
+		var bev = new NativeBufferEvent.socket(base, fd, bev_opt_close_on_free)
+		var addr=""
+		var conn = spawn_connection(bev, addr)
+		bev.enable ev_read|ev_write
+		bev.setcb conn
+	end
+
+	# Listen on domain & file for new connection, which will callback `spawn_connection`
+	fun bind_to_unix(hostname: String, file: String): nullable ConnectionListener
```

Comment:
Same here, the code seems pretty similar to the regular `bind_to`, can the two functions be merged?

Reply:
I prefer keeping it separated because of its use in `nitcorn::unix`, plus args are not the same type, a new indirection for bind_to would cause a shitstorm in nitcorn...